sdk/tests/conformance/buffers/buffer-data-and-buffer-sub-data.html

```diff
@@ -76,59 +73,80 @@
     wtu.glErrorShouldBe(gl, gl.INVALID_VALUE,
         "calling bufferData when buffer size is negative should generate INVALID_VALUE");
 
-    gl.bufferData(gl.ARRAY_BUFFER, 0, gl.STATIC_DRAW);
```

Comment:
I see. Sorry, I'd missed the test cases with new ArrayBuffer(0) and new ArrayBuffer(4).